lib/index.js

```diff
@@ -41,12 +41,16 @@ import {documentLoader as _documentLoader} from './documentLoader.js';
 export const defaultDocumentLoader =
   jsigs.extendContextLoader(_documentLoader);
 import * as credentialsContext from 'credentials-context';
+import * as contexts from './contexts/index.js';
 
 const {AuthenticationProofPurpose} = jsigs.purposes;
 const {constants: {CREDENTIALS_CONTEXT_V1_URL}} = credentialsContext;
 
 export {CredentialIssuancePurpose};
 
+// export for testing
+export default contexts;
```

Comment:
-1 to any `default` exports, always use named exports.

Reply:
Updated [57d87ae](https://github.com/digitalbazaar/vc-js/pull/115/commits/57d87aed52a7f60c9c03b942950e8562b7774d6e)